fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -1206,21 +1127,25 @@ public void testGetVersionResponseContentTypes() throws Exception {
         }
     }
 
-    @Ignore //TODO Fix this test
     @Test
     public void testDatetimeNegotiationLDPRv() throws Exception {
+        final String startDatetime = MEMENTO_RFC_1123_FORMATTER.format(Instant.now().atZone(ZoneOffset.UTC));
+
         final CloseableHttpClient customClient = createClient(true);
 
         createVersionedContainer(id);
-        final String memento1 =
-            MEMENTO_RFC_1123_FORMATTER.format(ISO_INSTANT.parse("2017-06-10T11:41:00Z", Instant::from));
         final String version1Uri = createLDPRSMementoWithExistingBody();
-            MEMENTO_RFC_1123_FORMATTER.format(ISO_INSTANT.parse("2016-06-17T11:41:00Z", Instant::from));
-        final String version2Uri = createLDPRSMementoWithExistingBody();
+
+        TimeUnit.SECONDS.sleep(1);
 
         // Request datetime between memento1 and memento2
-        final String request1Datetime =
-            MEMENTO_RFC_1123_FORMATTER.format(ISO_INSTANT.parse("2017-01-12T00:00:00Z", Instant::from));
+        final String request1Datetime = MEMENTO_RFC_1123_FORMATTER.format(Instant.now().atZone(ZoneOffset.UTC));
+
+        TimeUnit.SECONDS.sleep(1);
+
+        final String version2Uri = createLDPRSMementoWithExistingBody();
```

Comment:
Hmm... Yes, I'm confused now too. I'll see what's going on here tomorrow.

Reply:
Ah. So, what's going on here is that the most recent version of the resource is already tagged with a memento (ie there is no mutable head); so a new memento is not created. Instead, the service returns the location of the most recent memento.

Reply:
I'll fix the test so that it's testing what it's supposed to be testing, but it's an open question if you're alright with the behavior I just described.